content/en/users/im/_index.md

```diff
@@ -19,12 +19,14 @@ It is a service that features a web-based GUI, a XML-RPC API, a REST API and a
 command-line interface. For detailed information about Infrastructure Manager
 please visit its [documentation pages](https://imdocs.readthedocs.io).
 
-IM is integrated with the [EGI Check-in Service](../../check-in). A very easy
+IM is integrated with the [EGI Check-in Service](../check-in). A very easy
```

Comment:
(putting the comment here, but actually refers to line 6)

It would be good to change the description instead of "How to use IM" to something more like "Using Infrastructure Manager on EGI Cloud" (that's how we have it in EC3)

Reply:
Changed.

Reply:
It would be even better to use the label of a page in the left-side nav tree, the title and subtitle so that the users who do not know what IM is can still understand at a glance what this page will be about. E.g. in the left-side and the title to be "Automating Deployments" and the subtitle to be "Use Infrastructure Manager to automate VM deployment in the EGI Cloud".